staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2843,6 +2843,17 @@ type PodSpec struct {
 	// +patchMergeKey=name
 	// +patchStrategy=merge
 	Containers []Container `json:"containers" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,2,rep,name=containers"`
+	// EphemeralContainers is the list of ephemeral containers that run in this pod. Ephemeral containers
```

Comment:
Note that the name should be `ephemeralContainers` in Godoc when referencing the field.  I'm ok with this being a follow up if there's nothing else in this pass.